std_msgs/Header header
data_processing/DetectedObject[] objects
